app/src/main/res/layout/dashboard_list_item_category.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+        <import type="androidx.databinding.ObservableField"/>
+        <import type="androidx.lifecycle.LiveData"/>
+        <import type="org.cru.godtools.base.ui.util.ModelUtils" />
+        <import type="java.util.Locale" />
+
+        <variable name="category" type="String" />
+        <variable name="selectedCategory" type="LiveData&lt;String>" />
+        <variable name="callbacks" type="ObservableField&lt;org.cru.godtools.ui.dashboard.tools.CategoryAdapterCallbacks>"/>
+    </data>
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <androidx.cardview.widget.CardView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="4dp"
+            android:minHeight="64dp"
+            android:onClick="@{() -> callbacks.onCategorySelected(category)}"
+            android:padding="6dp"
+            app:cardBackgroundColor="@{category.equals(selectedCategory) ? @color/gray_80 : @color/white }"
+            app:cardCornerRadius="@dimen/card_corner_radius"
+            app:cardElevation="4dp"
+            tools:cardBackgrounColor="@color/gray_80">
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="20dp"
+                android:layout_marginTop="15dp"
+                android:layout_marginBottom="15dp"
+                android:text="@{ModelUtils.getToolCategoryName(category, context, Locale.getDefault())}"
```

Comment:
We will need to get clarification, but my guess is that you will want to use the primaryLanguage as the locale for the category name

Reply:
and if we aren't using `primaryLanguage`, then you will want to add a `@JvmOverloads` annotation to the method and not provide a specific locale to the method.